samcli/local/apigw/local_apigw_service.py

```diff
@@ -17,24 +17,27 @@
 
 class Route(object):
 
-    def __init__(self, methods, function_name, path, binary_types=None):
+    def __init__(self, methods, function_name, path, stage_name=None, binary_types=None):
         """
         Creates an ApiGatewayRoute
 
         :param list(str) methods: List of HTTP Methods
         :param function_name: Name of the Lambda function this API is connected to
         :param str path: Path off the base url
+        :param str stage_name: Name of the API stage
         """
         self.methods = methods
         self.function_name = function_name
         self.path = path
+        self.stage_name = stage_name
         self.binary_types = binary_types or []
 
 
 class LocalApigwService(BaseLocalService):
 
     _DEFAULT_PORT = 3000
     _DEFAULT_HOST = '127.0.0.1'
+    _DEFAULT_STAGE = 'Prod'
```

Comment:
Implicit APIs default to Prod if they don't reference an AWS::Serverless::API resource.